stdlib/string.ml

```diff
@@ -231,3 +237,51 @@ let to_seq s = bos s |> B.to_seq
 let to_seqi s = bos s |> B.to_seqi
 
 let of_seq g = B.of_seq g |> bts
+
+(** {6 Binary encoding/decoding of integers} *)
+
+(* These are all duplicated in bytes.ml *)
+
+external get_uint8 : string -> int -> int = "%string_safe_get"
+external get_uint16_ne : string -> int -> int = "%caml_string_get16"
+external get_int32_ne : string -> int -> int32 = "%caml_string_get32"
+external get_int64_ne : string -> int -> int64 = "%caml_string_get64"
```

Comment:
The runtime already has the correctly-named primitives for the `string_` versions (the names of these four primitives constitute the only difference in the code between the `String` and `Bytes` versions)